client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md

```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
 ---
 ## Introduction to the Information Security and Quality Assurance Projects
 
-This introduction is a stub
+So, now it’s time to put your newly learnt skills to work! By working on these projects, you will have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```

Comment:
```suggestion
Now it’s time to put your newly learnt skills to work! By working on these projects, you would have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```

Reply:
Please don't use the word `learnt`.  That is British English and it should be the American English word `learned`.

Reply:
Yes, I have made the cahnges.

Reply:
I don't see the changes though

Reply:
I updated the file can you check this one? @thecodingaviator